frontend_lib/src/component/ShareLink/ShareLink.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { translate } from 'react-i18next'
+import Radium from 'radium'
+import ComposedIcon from '../Icon/ComposedIcon.jsx'
+
+class ShareLink extends React.Component {
+  handleCopyToClipboard = () => {
+    let tmp = document.createElement('textarea')
+    document.body.appendChild(tmp)
+    tmp.value = this.props.link
+    tmp.select()
+    document.execCommand('copy')
+    document.body.removeChild(tmp)
+    /* or https://stackoverflow.com/questions/36639681/how-to-copy-text-from-a-div-to-clipboard See what's best */
```

Comment:
it the right way to link to the source of a non intuitive snippet copied from the web.
But you must use the comment convention we use, which would be, for you:
`// INFO - GB - 2019-07-17 - the comment`